workflow/sync/semaphore.go

```diff
@@ -175,11 +186,10 @@ func (s *PrioritySemaphore) tryAcquire(holderKey string) (bool, string) {
 	// If it is not a front key, it needs to wait for its turn.
 	if s.pending.Len() > 0 {
 		item := s.pending.peek()
-		nextKey = fmt.Sprintf("%v", item.key)
```

Comment:
item.key is already a string, so I didn't understand the purpose of `fmt.Sprintf("%v", item.key)`